pages/box_buster.vue

```diff
@@ -45,26 +45,30 @@
         <b-row>
           <b-col>
             <ul>
-              <li>Total of {{ total }} {{ plural_ize('plate', total) }} in box</li>
+              <li>Total of {{ total }} {{ 'plate' | pluralize(total) }} in the box</li>
```

Comment:
The pluralizing in this file is a bit confusing, using the `inclusive` flag, like from [here](https://www.npmjs.com/package/pluralize#usage) might help?
e.g `Total of {{ pluralize('plate', total, true) }} in the box`